app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -76,6 +78,10 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
             showPleaseRateDialog();
         }
 
+        importFromCsv.setText(preferencesManager.isOnFreeVersion() ?
```

Comment:
Instead of doing this in onCreate(), let's do this in onResume(). This way, when the users returns from the "Buy Premium" page having bought premium, the UI will automatically update. It will do redundant text loads, but I don't know a great way around that minus some sort of EventBus or singleton pub/sub hack. :/